src/dh-gex.c

```diff
@@ -58,6 +59,14 @@ static struct ssh_packet_callbacks_struct ssh_dhgex_client_callbacks = {
     .user = NULL
 };
 
+int min(int a, int b) {
```

Comment:
C99 doesn't declare a min/max function anywhere. So we do that here and then let the compiler inline them.